packages/terra-form-select/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Changelog
 
 ## Unreleased
+* Added
+  * Added `aria-label` to list options.
+
+* Changed
+  * Changed `hideRequired`, `isLabelHidden`, `onChange` props to private.
+  * Updated aria attributes.
```

Comment:
Can this be a little more descriptive?

Reply:
https://github.com/cerner/terra-core/pull/3707/commits/4700debfb340ff2e8b654e94a9fb801accb3bdd0